blog/_posts/2016-09-16-juliacon2016.md

```diff
@@ -0,0 +1,154 @@
+I saw that Professor Holy is widely admired in the entire Julia community due to his involvement
+in various packages and the key issues on the language. I noticed that he asked some pretty neat
+insightful questions at various earlier sessions too. Stefan was up next with his super-important
+Julia 1.0 [talk](https://www.youtube.com/watch?v=5gXMpbY1kJY). It was quite a comprehensive list
+of things that needed to be done before Julia got 1.0 ready and he touched on a variety of areas
+such as the compiler, the type system, the runtime, multi-threading, strings and so on.
+
+The next session saw a team from UC Berkeley show off their
+[autonomous racing car](https://www.youtube.com/watch?v=bX4TXWO7dA0) that uses some optimization
+packages (JuMP and Ipopt in particular) to solve real-time optimization problems. Julia was running
+on an ARM chip with Ubuntu 14.04 installed. Julia can also run on the Raspberry Pi, and my colleague
+Avik took some time to [show off](https://www.youtube.com/watch?v=EvJ-OvTC5eE) a cool Minecraft demo
+running on the Pi. The talk after that was about JuliaBox. Nishanth, another colleague of mine, has
+been hard at work porting JuliaBox to Google Cloud from AWS, and he
+[spoke](https://www.youtube.com/watch?v=j0tmyWJ-aSQ) about his exciting plans for JuliaBox.
+
+Post lunch, I had to choose again between parallel sessions, but I couldn’t quite
+resist the session with stochastic PDEs and Finite Elements. Kristoffer Carlsson
+reviewed the state of [FEM in Julia](https://www.youtube.com/watch?v=30TUEhbGmuc),
+talking about the packages and ecosystem for every FEM step from assembly to the
+conjugate gradient. The next [talk](https://www.youtube.com/watch?v=EEP2NMgC9Zo)
+was given by a professor at TU Vienna whose group conducts research on nano-biosensors,
+and the group uses Julia to solve the stochastic PDEs that come up when trying to model
+noise and fluctuations. The next [talk](https://www.youtube.com/watch?v=IjJqVwtWO3s)
+on astrodynamics was very interesting in that it gave me an insight into the kinds of
+computational challenges faced by scientists in the field. There were also some interesting
+demos which I enjoyed, particularly the one where we modelled and visualized a target orbit,
+which superimposed upon a visual of the earth in space.
+
+In the afternoon, after much consideration, I went to the session that featured statistical
+modelling and least squares. The first talk on
+[sparse least squares optimization](https://www.youtube.com/watch?v=S5sA-Ch_KPo) problems
+gave me a flavor of the kinds of models and problems economists need to solve, and how the
+Julia ecosystem helps them. The [next talk](https://www.youtube.com/watch?v=ZfjRjljXYXk)
+on computational neuroscience focussed on dealing with tens of terabytes of brain data
+coming from both animals and human surgery patients. I had a very interesting discussion
+with John earlier about his work, and I could get a keen sense of how why the package
```

Comment:
Sentence flow fix: `...and I was able to get a keen sense of how the package....`